components/Card/AttributeOptionKeyValues.vue

```diff
@@ -5,19 +5,19 @@
 <template>
     <div :class="['options', {'options--disabled': disabled}]">
         <div
-            v-for="(key, index) in optionKeys"
+            v-for="(key, index) in Object.keys(options)"
```

Comment:
It's better to do computed from that - every time anything in template will render, it will be calculated again